app/V1Module/presenters/AssignmentsPresenter.php

```diff
@@ -182,16 +207,17 @@ public function actionUpdateDetail(string $id) {
       $this->assignmentEmailsSender->assignmentCreated($assignment);
     }
 
-    // retrieve localizations and prepare some temp variables
-    $localizedTexts = $req->getPost("localizedTexts");
-    $localizations = [];
-
-    // localized texts cannot be empty
-    if (count($localizedTexts) == 0) {
-      throw new InvalidArgumentException("No entry for localized texts given.");
+    // if points were changed, go through all submissions and recalculate points
+    if ($oldFirstDeadlinePoints != $firstDeadlinePoints ||
+        $oldSecondDeadlinePoints != $secondDeadlinePoints) {
```

Comment:
Check also for changes in the deadlines themselves and change in pointsPercentualThreshold.

Reply:
On second thoughts, where are the deadlines applied? When assigning the points or when the best submission of a particular assignment is being determined to display final points? If they are not needed for the points recalculation, then there is no need to test their change...

Reply:
We need to check this too and recalculate even for deadlines... Points are assigned according to this information too.